### ISSUE TITLE
Candle 1.2b

### ISSUE THREAD

**Issue by wryan67** [pihnat/rpi-Candle#11]
Please update the rpi-Candle to 1.2b.  There are lots of new features to take advantage of.  :)

**Comment by pihnat** [comment]
Hi,
I hadn't updated to 1.2b because Denvi's site says 1.2b is "experimental" so I was waiting until it became the new version. However I notice that there hasn't been any updates to the source code since 2019 so maybe this might end up being the final version. I don't have time at the moment to try porting 1.2b to the raspi. It's on my list of things to do but won't happen anytime soon unfortunately.

**Comment by wryan67** [comment]
I agree, I don't think Denvi will add new features or even break fixes for that matter.  I've used 1.2b on windows and it is solid, so I say, let's go for it on the RPi!  I have a couple of small machines and it's so much easier to just plug an RPi into them and use vnc to remote connect, rather than carting my laptop around.  I've also tried bCNC and don't care for their interface.  Candle is so much cleaner and easier to read.  Thank you in advance for your work converting the Denvi project to RPi!
